### ISSUE TITLE
what is the difference with adding l1 or KL-loss to final loss function ? 

### ISSUE THREAD

**Issue by le02146** [abhigoogol/Autoencoders-using-Pytorch#1]

**Comment by abhisheksambyal** [comment]
L1 loss is used to compute pixel wise error between two images where as KL divergence is used to measure the difference between two probability distributions.
